fn_apility/fn_apility/util/selftest.py

```diff
@@ -0,0 +1,21 @@
+# (c) Copyright IBM Corp. 2010, 2018. All Rights Reserved.
+# -*- coding: utf-8 -*-
+# pragma pylint: disable=unused-argument, no-self-use
+"""Function implementation
+   test with: resilient-circuits selftest -l fn_apility
+"""
+
+import logging
+
+log = logging.getLogger(__name__)
+log.setLevel(logging.INFO)
+log.addHandler(logging.StreamHandler())
+
+
+def selftest_function(opts):
+    """
+    Placeholder for selftest function. An example use would be to test package api connectivity.
+    Suggested return values are be unimplemented, success, or failure.
+    """
+    options = opts.get("fn_apility", {})
```

Comment:
This should make an api call which confirms that access to this end-point is available for function calls. is there a simple api call which can be coded to ensure connectivity is available?

Reply:
So, I have done necessary changes but somehow couldn't run it, plz have a look on that.

Reply:
see my comment in slack